python/docs/source/development/logger.rst

```diff
@@ -52,7 +52,7 @@ Example log entry:
         "file": "/path/to/file.py",
         "line_no": "17",
```

Comment:
What about line_no?

Reply:
This indicates the line number retrieved from stack frame by using `frame.lineno`.
On second thought maybe we should use `lineno` or `line_number` instead `line_no`?

Reply:
yeah

Reply:
Sounds good! Let me submit a quick followup for https://github.com/apache/spark/pull/47145 as I think this is technically not related to this PR.

Reply:
Created followup here: https://github.com/apache/spark/pull/47437